code/datums/id_trim/centcom.dm

```diff
@@ -23,6 +23,8 @@
 	department_color = COLOR_CENTCOM_BLUE
 	subdepartment_color = COLOR_SERVICE_LIME
 	big_pointer = FALSE
+	honorifics = list("Custodian")
+	honorific_positions = HONORIFIC_POSITION_LAST | HONORIFIC_POSITION_NONE
```

Comment:
I noticed every single job has `HONORIFIC_POSITION_NONE` set. 

Do you intend on there being jobs that aren't allowed to disable their honorific? 
Or rather do you think there are usecases where a job would be forced to use their honorific? 

If no, I feel like we should just axe the flag and manually include "Disable Honorific" in the input list in ID card code. 
It'd make it a lot easier to write if people could just slap in the flags they want.

Reply:
Alternatively you could include "Disable Honorific" by default and instead have a flag `HONORIFIC_POSITION_FORCED` which disables disable honorific

Reply:
> I noticed every single job with a position set also has `HONORIFIC_POSITION_NONE` set.
> 
> Do you intend on there being jobs that aren't allowed to disable their honorific? Or rather do you think there are usecases where a job would be forced to use their honorific?
> 
> If no, I feel like we should just axe the flag and manually include "Disable Honorific" in the input list in ID card code. It'd make it a lot easier to write if people could just slap in the flags they want.

Honestly, I had planned on making it to be a default option so it wouldn't always need to be concluded. After a while though, the potential of a forced honorific felt like a fun idea that could be capitalized on by someone more creative than I. I think I left a comment in the code somewhere mentioning that.

Reply:
Then how do you feel about my follow up idea, where you make honorifics have "disableable" in built and you have a separate flag for forced? 
```dm
	honorifics = list("Custodian")
	honorific_positions = HONORIFIC_POSITION_LAST // no none flag, it is implied - you can disable it if you want
```
```dm
	honorifics = list("Judge")
	honorific_positions = HONORIFIC_POSITION_LAST | HONORIFIC_FORCED // force flag meaning you cannot turn it off
```
```
var/list/honorific_options = list()
if(!(honorific_positions & HONORIFIC_FORCED))
    honorific_options += "Disable"
```
IMO this would make it easier to work with in the future but I will leave it up2u

Reply:
I'm fine with just leaving it as-is for now. Double the technical debt and give it to the next person.